tests/plugins/test_libplugin.c

```diff
@@ -106,6 +106,14 @@ static const struct plugin_command commands[] = { {
 		"Makes a simple getinfo call, to test rpc socket.",
 		"",
 		json_testrpc,
```

Comment:
What does the C standard say about omission of a `struct` field in an initializer?  Does it get set to 0, or to undefined behavior?

Reply:
The `struct` is static storage duration, so yes, any fields not specified in the initializer should be 0, and fortunately `false` is 0 as well.